pkg/dockerutil/dockerutils.go

```diff
@@ -45,6 +46,19 @@ import (
 // NetName provides the default network name for ddev.
 const NetName = "ddev_default"
 
+// DockerVersionConstraint is the current minimum version of Docker recommended for DDEV.
```

Comment:
```suggestion
// DockerVersionConstraint specifies the Docker server version required for DDEV.
```

Reply:
Strictly speaking, we don't need a `DockerVersionConstraint`, `DockerAPIVersionMinimum` should be enough.

It's not required now, see my comment below.

Reply:
It's true that we're probably doing too much in too many places.

Reply:
Let's remove `DockerVersionConstraint`, one check on `ddev start` should be enough.